src/machine/board_nrf52840_generic.go

```diff
@@ -0,0 +1,27 @@
+//go:build nrf52840 && nrf52840_generic
+
+package machine
+
+const HasLowFrequencyCrystal = true
```

Comment:
This is not a given, some boards in fact don't have a low frequency crystal (though this is rare).
But maybe it's rare enough that we can leave it as-is.

Reply:
I refactored this a bit so that it configurable via build flag.